examples/jsm/exporters/GLTFExporter.js

```diff
@@ -927,10 +942,25 @@ class GLTFWriter {
 
 		}
 
+		console.warn( 'THREE.GLTFExporter: Merged metalnessMap and roughnessMap textures.' );
+
 		return texture;
 
 	}
 
+
+	async decompressTexture( texture, maxTextureSize = Infinity ) {
```

Comment:
It would be interesting to standardize the `async` functions with the sufix `Async` like, `renderer.hasFeatureAsync()`, `renderer.computeAsync`, ...

Reply:
I wonder if we need this if there is only one version of the method. Another point to think about: If the API gets a more async nature over time and we want to promote this style, I wonder if a default name like `hasFeature()` shoul d be async and the sync version `hasFeatureSync()`.

Reply:
It's an interesting point. I think asynchronous calls will still be the exception, I think that most of the code are related to objects manipulation and TSL, in this sense the code would always be synchronous preferably. Asynchronous functions are linked to processes that derive a working time or initialization.

Reply:
Not a strong preference, but I do tend to like the `*Async` suffix for async methods, at least when they are the exception not the norm.